assembl/static2/js/app/utils/translations.js

```diff
@@ -118,8 +118,7 @@ const Translations = {
       existingUsername: 'Le nom d’utilisateur que vous avez renseigné existe déjà',
       existingEmail: 'L’e-mail que vous avez renseigné existe déjà',
       emailNotFound: 'L’e-mail que vous avez renseigné n’a pas été trouvé. Veuillez réessayer.',
-      incorrectPasswordLogin:
-        'Le mot de passe que vous avez entré ne correspond pas avec l’utilisateur saisi. Veuillez réessayer.',
+      incorrectPasswordLogin: 'Le mot de passe que vous avez entré ne correspond pas avec l’utilisateur saisi. Veuillez réessayer.',
```

Comment:
Oh, I know there is no real change here, but it's a bad security practice to divulge that the login actually exist! I can't find where this message is used in the code, maybe it's not used anymore. If this is the case, we should remove it.